### DISCUSSION TITLE
`slave` status not showing

### DISCUSSION THREAD

**Discussion by kjoconnor** [revk/xESP32-Faikout#328] [General]
I see that there's a `slave` bool available to show if the unit is unable to perform the selected function, but I don't see that in my MQTT messages. Is it possible my model (CTXS07WVJU9) does not report that via S21? Or is there some switch to flip to make this available?

**Comment by revk** [comment]
I'll have to check, that may be only X50A

**Comment by revk** [comment]
Sorry, slave is only X50A.

If you know of any bit in S21 status that provides this, let me know.

**Comment by josegonzalez** [comment]
@kjoconnor just checking in, when you installed the faikin on your CTXS07WVJU9, did you have to purchase S21 cables or was that already available in the unit? Any chance you have pictures of the installation?
